lib/torrent.js

```diff
@@ -1084,7 +1084,7 @@ class Torrent extends EventEmitter {
     })
 
     wire.bitfield(this.bitfield) // always send bitfield (required)
-    wire.uninterested() // always start out uninterested (as per protocol)
+    this._updateWireInterest(wire) // set inital interest
```

Comment:
If you support the FAST extensions then you should also update the interested state when a `HaveAll` message is received.

Reply:
I don't think webtorrent supports BEP6? https://github.com/webtorrent/bittorrent-protocol (sounds like a good thing to make an issue about though)

`_updateWireInterest()` is also called anytime a selection is made (piece priority changed), and also when a piece is successfully validated or added to the store from `_checkdone()` (because it calls `_gcSelections()`, which calls `_updateSelections()`)